### PR TITLE
add campaign

### PR DESCRIPTION
**Pull request by peach0777**
### Changes Made

- Created a CampaignController
- Created a Register trait
- Created CampaignRequest
- Added budget main blade with modal
- Added modal js and css
- Added  campaign routes
- Passed an array to the include field for sidebar to determine active nav

### CODE REVIEW

File: public/js/modal.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Get the modal
+var modal = document.getElementById("myModal");
```
Comment (ChronoDevs) [inline public/js/modal.js]: for javascript it's good to use single quote

Comment (ChronoDevs) [reply public/js/modal.js]: When you consistently use single quotes, you can avoid escaping the single quotes within your strings, which can make the code cleaner.

Comment (peach0777) [reply public/js/modal.js]: Understood, sir.

Comment (peach0777) [reply public/js/modal.js]: Thank you for the information, sir.

**Pull request merged**